RebbleOS/vibrate.c

```diff
@@ -48,70 +67,97 @@ void vibrate_init(void)
     _vibrate_queue = xQueueCreate(1, sizeof(uint8_t));
```

Comment:
1 queue depth, intentional? Do queued commands have to wait for queue, or immediately fail?